parallel_hashmap/phmap.h

```diff
@@ -1806,7 +1806,7 @@ class raw_hash_set
 
     size_t bucket_count() const { return capacity_; }
     float load_factor() const {
-        return capacity_ ? static_cast<double>(size()) / capacity_ : 0.0;
+        return capacity_ ? static_cast<float>(size()) / capacity_ : 0.0f;
```

Comment:
Would this work?

```suggestion
        return capacity_ ? static_cast<float>(static_cast<double>(size()) / capacity_) : 0.0f;
```

Reply:
Yes, seems to do the trick.

Reply:
What compiler do you see this warning on that line? I don't see with with clang-17 or gcc-12.

Reply:
I'm using GCC 11.4 with -Wfloat-conversion.